docs/btc-methods.md

```diff
@@ -0,0 +1,73 @@
+# BTC Methods
+
+Here we document the Bitcoin methods that an account Snap may implement to
+support requests originated from dapps.
+
+## btc_sendmany
+
+This method is similar to the `sendmany` RPC method from Bitcoin Core, but its
+parameters are passed in an object instead of an array, and are named in
+camelCase. Also, dummy values aren't allowed.
+
+### Parameters
+
+- **Request object (required):**
+  - Type: `object`
+  - Properties:
+    - `amounts`
+      - Description: A JSON object with recipient addresses and amounts.
+      - Type: `object`
+      - Properties:
+        - `[key]: string`: Address of the recipient
+        - `[value]: string`: Amount to send to the recipient in BTC
+    - `comment` (optional)
+      - Description: A comment.
+      - Type: `string`
+    - `subtractFeeFrom` (optional)
+      - Description: The fee will be equally deducted from the amount of each
+        selected address. Those recipients will receive less bitcoins than you
+        enter in their corresponding amount field. If no addresses are specified
+        here, the sender pays the fee.
+      - Type: `array`
+      - Properties:
+        - Type: `string`
+    - `replaceable` (optional)
```

Comment:
Sure, it's better explained here:
- <https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki>
- <https://learnmeabitcoin.com/technical/transaction/#structure-inputs-sequence>

Reply:
Thanks for checking @stanleyyconsensys, lets use the Core implementation as reference for the sequence value in case this option is set.

We might not support it in the first version since it requires bumpfee, but we should work toward supporting it.